src/tools.py

```diff
@@ -603,21 +603,23 @@ def tools_shell(command=None):
         shell.interact()
 
 
-def tools_basic_space_cleanup():
+def tools_basic_space_cleanup(days=90):
     """
     Basic space cleanup.
 
     apt autoremove
     apt autoclean
     journalctl vacuum (leaves 50M of logs)
     archived logs removal
+    yunohost logs removal (older than 90 days by default)
     """
     subprocess.run("apt autoremove && apt autoclean", shell=True)
     subprocess.run("journalctl --vacuum-size=50M", shell=True)
     subprocess.run("rm /var/log/*.gz", shell=True)
     subprocess.run("rm /var/log/*/*.gz", shell=True)
     subprocess.run("rm /var/log/*.?", shell=True)
     subprocess.run("rm /var/log/*/*.?", shell=True)
+    subprocess.run(f"find /var/log/yunohost/operations/ -type f,l -mtime +{days} -execdir rm {} +", shell=True)
```

Comment:
Sounds like Python is not happy about the empty `{}` because it's a f-string ?

`SyntaxError: f-string: valid expression required before '}'`